soda-core/src/soda_core/common/sql_dialect.py

```diff
@@ -28,6 +29,11 @@ def quote_default(self, identifier: Optional[str]) -> Optional[str]:
             else None
         )
 
+    def build_fully_qualified_sql_name(self, dataset_identifier: DatasetIdentifier) -> str:
```

Comment:
wouldn't it make sense to make this a method on the dataset identifier for more portability as we discussed?

Reply:
makes sense, but that has to be done in a lot of places.  planned to do that later all in 1 go.  ok?